src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,53 @@
-import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type MovieCardProps = {
+  card: Movie;
+};
+
+export const MovieCard: React.FC<MovieCardProps> = (
+  { card }: MovieCardProps,
+) => {
+  return (
```

Comment:
you can use destructuring here -> `{ imgUrl, title, description, imdbUrl } = card;` (or `movie` if you change name of prop)